internal/cloud/realm/app.go

```diff
@@ -83,20 +84,31 @@ func (c *client) DeleteApp(groupID, appID string) error {
 
 // AppFilter represents the optional filter parameters available for lists of apps
 type AppFilter struct {
-	GroupID string
-	App     string // can be client app id or name
+	GroupID  string
+	App      string // can be client app id or name
+	Products []string
 }
 
+const (
+	productEmpty    = ""
```

Comment:
[nit] does this really need to be a const?

Reply:
no, i even forgot to use it down below.  i'll remove